src/main/java/org/commcare/formplayer/util/FormplayerDatadog.java

```diff
@@ -234,4 +237,11 @@ private String getTagValueToSend(String tagName, String tagValue) {
         }
     }
 
+    public static void handleKeepDropAPMTraces(boolean keepAPMTrace) {
+        Span span = GlobalTracer.get().activeSpan();
+        String keepDropTag;
+        keepDropTag = keepAPMTrace ? DDTags.MANUAL_KEEP : DDTags.MANUAL_DROP;
+        span.setTag(keepDropTag, true);
```

Comment:
this can land us in a scenario where both `MANUAL_KEEP` and `MANUAL_DROP` are set to `true` which seems a bit error-prone. I would modify this code as - 
````
span.setTag(MANUAL_KEEP, keepAPMTrace)
span.setTag(MANUAL_DROP, !keepAPMTrace)
````